tasks/pages/templates.tsx

```diff
@@ -78,15 +78,15 @@ export const TaskTemplateContext = createContext<TaskTemplateContextType>({
 
 const PersonalTaskTemplatesPage: NextPage = ({ language }: PropsWithLanguage<PersonalTaskTemplateTranslation>) => {
   const translation = useTranslation(language, defaultPersonalTaskTemplateTranslations)
-  const router = useRouter()
-  const { templateId } = router.query
+  const templateId = useRouteParameters<never, 'templateId'>().templateId
   const [usedQueryParam, setUsedQueryParam] = useState(false)
   const { user } = useAuth()
   const { isLoading, isError, data } = usePersonalTaskTemplateQuery(user?.id)
 
   const [contextState, setContextState] = useState<TaskTemplateContextState>(taskTemplateContextState)
 
-  const createMutation = useCreateMutation('personalTaskTemplates', taskTemplate =>
+  const wardId = '' // TODO: why is this empty? (tracing back where is value originally came from lead me to the fact that this value would always be undefined or an empty string, thus I hardcoded it here now but it would be great to know if this is correct)
```

Comment:
@DasProffi

Reply:
There should be no need for a ward id in this view as these are personal templates. But the ward id from the router.query is used in the TaskTemplateDisplay.tsx component